roles/growpart/tasks/centos-7.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: install epel-release repo
```

Comment:
EPEL is already installed in preinstall section. Just make this role depend on it.